command/runner.go

```diff
@@ -124,10 +107,11 @@ func WithLocalUser(user string) func(*LocalRunner) {
 	}
 }
 
-func NewLocalRunner(e config.CommonEnvironment, options ...func(*LocalRunner)) *LocalRunner {
+func NewLocalRunner(e config.CommonEnvironment, isWindows bool, options ...func(*LocalRunner)) *LocalRunner {
```

Comment:
`LocalRunner` is, well, local, so you can rely on `GOOS`.

Reply:
I have injected the osCommand.